packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -1370,6 +1378,21 @@ test('transfer vault', async t => {
     'transfer closed old notifier',
   );
 
+  t.deepEqual(
+    {
+      // simplify test by including any properties not mentioned below
+      ...inviteProps,
+      debtSnapshot: {
+        debt: debtAmount,
+        interest: aliceFinish.value.debtSnapshot.interest,
+      },
+      description: 'TransferVault',
+      locked: collateralAmount,
+      vaultState: 'active',
+    },
+    inviteProps,
+  );
+
```

Comment:
The actual value goes first. Also [Ava has a `like`](https://github.com/avajs/ava/blob/main/docs/03-assertions.md#likeactual-selector-message):
```suggestion
  t.like(
    inviteProps,
    {
      debtSnapshot: {
        debt: debtAmount,
        interest: aliceFinish.value.debtSnapshot.interest,
      },
      description: 'TransferVault',
      locked: collateralAmount,
      vaultState: 'active',
    },
  );

```